src/airport.js

```diff
@@ -4,22 +4,24 @@ const Weather = require(`./weather`)
 class Airport {
   constructor(capacity = 0) {
```

Comment:
I would not use the constructor method to override the capacity, instead I would create an overriding capacity function in order to make the capacity setting less reliant on the constructor of the airport class. If more code was added to the constructor, it would make it overly complicated and if tests failed, this would be a problem.